ts-apps/decision-logic-visualizer/src/lib/data/viz-expr-to-lir.ts

```diff
@@ -127,46 +128,48 @@ function transform(
         env,
         neg.negand
       )
+
```

Comment:
The stuff that's not the App case is just refactoring to make the transformation code more readable and understandable